plexapi/myplex.py

```diff
@@ -698,7 +707,7 @@ def delete(self):
         self._server.query(key, self._server._session.delete)
 
 
-def _connect(cls, url, token, timeout, results, i):
+def _connect(cls, url, token, timeout, results, i, connected_event=None):
```

Comment:
What is connected_event used for, can you add a comment in the doc string here about what what is is?